Kronos-Shifts-Connector/Microsoft.Teams.Shifts.Integration/Microsoft.Teams.Shifts.Integration.API/Common/AppSettings.cs

```diff
@@ -242,6 +242,11 @@ public string TenantId
         /// </summary>public string OrgJobPath => this.configuration["OrgJobPath"];
         public string SyncToNextDays => this.configuration["SyncToNextDays"];
 
+        /// <summary>
+        /// Gets the number of days in the future to sync swap shift eligiblity for.
+        /// </summary>
+        public string FutureSwapEligibilityDays => this.configuration["FutureSwapEligibilityDays"];
```

Comment:
What was the decision on finding eligible shifts for the same day, do we want to allow swaps on the same day?. We could configure it in appsetting with a setting like: bool AllowSameDaySwap

Reply:
Kronos doesn't let you do it if the shift has started but does if it hasn't so as it stands we're matching the Kronos functionality. 
Don't think we want to prevent same day if Kronos doesn't.

Reply:
Yeah okay, should we raise this in the meeting just to clarify?

Reply:
Sure